### DISCUSSION TITLE
Try out the new live-demo

### DISCUSSION THREAD

**Discussion by marcantondahmen** [automadcms/discussions#3] [Announcements]
From now on it will be possible to try the latest Automad release in a live-demo right in the browser. With only a few clicks and without any sign-up or similar obstacles. No local installation required anymore. 

[try.automad.org](https://try.automad.org)

Everyone gets a personal demo that will expire after one hour. In order to keep hosting cost as low as possible, all demos run on minimal hardware. So it might be possible that things feel slow when the demo server is very crowded.

Reporting issues is appreciated as usual.

Happy testing! 🙂

**Comment by Imerion** [comment]
Tried it yesterday and it worked great. Did not feel slow at all and the new version 2 is great!
